<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>TimeAdd</class>
    <widget class="QDialog" name="TimeAdd">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Time Add</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_Main">
            <item>
                <widget class="QLabel" name="label">
                    <property name="font">
                        <font>
                            <pointsize>12</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>Please enter the amount of time to skip forward:</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_Columns">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_Days">
                            <item>
                                <widget class="QLabel" name="lbl_Days">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Days</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="lineEditDays">
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_1Day">
                                    <property name="text">
                                        <string>+1 Day</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_7Days">
                                    <property name="text">
                                        <string>+7 Days</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_Days">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_Hours">
                            <item>
                                <widget class="QLabel" name="lbl_Hours">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Hours</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="lineEditHours">
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_1Hour">
                                    <property name="text">
                                        <string>+1 Hour</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_6Hours">
                                    <property name="text">
                                        <string>+6 Hours</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_12Hours">
                                    <property name="text">
                                        <string>+12 Hours</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_Hours">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_Minutes">
                            <item>
                                <widget class="QLabel" name="lbl_Minutes">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Minutes</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="lineEditMinutes">
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_5Mins">
                                    <property name="text">
                                        <string>+5 Mins</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_15Mins">
                                    <property name="text">
                                        <string>+15 Mins</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_30Mins">
                                    <property name="text">
                                        <string>+30 Mins</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_Minutes">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_Seconds">
                            <item>
                                <widget class="QLabel" name="lbl_Seconds">
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="text">
                                        <string>Seconds</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="lineEditSeconds">
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btn_30Secs">
                                    <property name="text">
                                        <string>+30 Secs</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_Seconds">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer_Bottom">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
                    </property>
                    <property name="centerButtons">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>TimeAdd</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>TimeAdd</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
